apps/common/js/components/leap-hands-view.jsx

```diff
@@ -56,19 +76,22 @@ export default class LeapHandsView extends React.Component {
     camera.position.fromArray([0, 500, 400]);
     camera.lookAt(new THREE.Vector3(0, 0, 0));
     scene.add(camera);
-    return {scene, camera};
+    return { scene, camera };
   }
 
   render() {
-    const { width, height } = this.props;
+    let width = this.width;
+    let height = this.height;
+    if (this.props.width != null){
+      width = this.props.width;
+      height = this.props.height;
+    }
```

Comment:
On the Seasons model, the switching of the views required that the renderer be resized each time the view was switched. On that model, I have the sizes of each viewport set via css, and each viewport has a different aspect ratio of width to height. I found the order in which resizing happened was tricky - needed to make sure that the containers were resized as well as the renderer viewport. 

The other issue I had (leading to the "make it square" was if I just used the size of the main container the hands would end up being stretched (on Seasons, the main window is wider than it is tall), and while I could switch the views and change the renderer viewport between each of the season panes, the initial rendering call fixed the aspect ratio of the hands, so they would retain the incorrect stretched appearance. 

Certainly the above image is incorrect and I agree on the other points - I'll see what I can do.